lib/forcex/client.ex

```diff
@@ -61,6 +61,20 @@ defmodule Forcex.Client do
     services = Forcex.services(client)
     %{client | services: services}
   end
+  
+  def create_sobject(client \\ %__MODULE__{}, name \\ "SOBject", map \\ %{})
+
+  def create_sobject(client, name, map) when is_atom(name) do
+    name = name 
+    |> Atom.to_string
+    |> String.capitalize
+
+    client
+    |> create_sobject(name, map)
+  end
+  def create_sobject(client, name, map) do
+    Forcex.post("/services/data/v20.0/sobjects/#{name}", map, client)
```

Comment:
@sebbean the reason that I have `client` as the trailing param is that for most of the calls, it is the least relevant to the name of the function. True, moving `client` to the first parameter would make it more cases of using `|>` possible, but `client` isn't the focus of many of the functions, nor is it transformed by many of the functions (the obvious exception is `Forcex.Client`).